### PR TITLE
add async_nbgrader package to docker image

### PR DESCRIPTION
**Pull request by rupeshparab**
Related to https://github.com/IllumiDesk/async_nbgrader/pull/1

### CODE REVIEW

**Comment by rupeshparab** [comment]
Test is failing because we need to create a new release after merging https://github.com/IllumiDesk/async_nbgrader/pull/2

**Review by jgwerner — APPROVED** [review]
LGTM

File: illumidesk-notebook/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-https://github.com/IllumiDesk/nbgrader/archive/v0.64.3.zip
+https://github.com/IllumiDesk/async_nbgrader/archive/v0.1.0.zip
```
Comment (jgwerner) [inline illumidesk-notebook/requirements.txt:1]: @rupeshparab update to v0.1.1 based on the [latest release](https://github.com/IllumiDesk/async_nbgrader).

Comment (rupeshparab) [reply illumidesk-notebook/requirements.txt]: updated release version

File: grader-notebook/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM illumidesk/illumidesk-notebook:aec555e49be6
```
Comment (rupeshparab) [inline grader-notebook/Dockerfile:1]: @jgwerner 
I had to change this static tag to the latest built tag of illumidesk-notebook/Dockerfile image for the tests to pass as the previous image didn't have our async_nbgrader package

Comment (jgwerner) [reply grader-notebook/Dockerfile:1]: @rupeshparab sounds good, we should probably default to latest anyway for dev and then we can set the image:tag version explicitly for our own deployments.

**Review by jgwerner — APPROVED** [review]
LGTM

**Pull request merged**